packages/contracts/src/deployConfigs.ts

```diff
@@ -43,10 +43,10 @@ async function networkConfigs(chainId: number): Promise<Config> {
           supply: parseUnits("10"),
         },
         ctndVesting: {
-          start: now + THIRTY_MIN,
+          start: now + THIRTY_MIN * 2,
         },
         ctndSale2: {
-          start: now,
+          start: now + THIRTY_MIN,
```

Comment:
why do we need the two sales for mandala? Want to test all of that?

Reply:
this is on hardhat, not mandala